src/service-bus-service.ts

```diff
@@ -0,0 +1,36 @@
+import { Guid } from 'guid-typescript';
+import { AzureWebCommandService } from './azure-web-command-service';
+
+export class ServiceBusService {
+    private baseUrl: string;
+    private sharedAccessKey: string;
+
+    constructor(sbNameSpace: string,
+                sharedAccesKey: string) {
+        this.baseUrl = sbNameSpace;
+        this.sharedAccessKey = sharedAccesKey;
+    }
+
+    public async sendMessageAsync(queueName: string, correlationId: Guid, args?: any[]): Promise<void> {
+        const url = `${this.baseUrl}/ReceiveWebEvent`;
```

Comment:
ReceiveWebEvent could be confusing from which point of view you look at it. It seems a little ambigious to me. Better would be something like /PostJob or /StartJob.

Reply:
Or PostCommand or SendCommand to keep it more consistent with the terminology below.

Reply:
I picked SendCommand